_dev/src/assets/json/translations/de/ui.json

```
@@ -9,6 +9,7 @@
       "help": "Hilfe"
     },
     "versionDoesNotExist": "Es scheint, dass die Version des Moduls nicht abgerufen werden kann. Es liegt an unserer Seite, bitte versuchen Sie es erneut.",
+    "moduleNeedActivation": "Marketing with Google is disabled, you will not be able to use any of it’s features until you enable your module.",
```

@m-Poulain can we have another PR to change the quote in the original string?

```suggestion
    "moduleNeedActivation": "Marketing with Google is disabled, you will not be able to use any of it's features until you enable your module.",
```